### PR TITLE
Add travis & add better tslint rules

### PR DESCRIPTION
**Pull request by Nasicus**
Travis currently only does the following:

- `npm run lint --bailOnLintError true`
- `npm run build -- --prod`

I didn't add an android build or even attempt to, because for me at the moment it's more important to ensure the code style & ensure the compilation / build works in general

I also updated `tslint.config` to match the one from `ark-explorer`.
I think it makes sense to have the same tslint in both project, since both are with angular 2+.
However I removed the following rules:
 - `"forin": true` ([source](https://palantir.github.io/tslint/rules/forin/))
   - reason: I don't like the enforcement of having to do the `hasOwnProperty` all the time
- `"radix": true` ([source](https://palantir.github.io/tslint/rules/radix/))
  - reason: find it unnecessary, never heard of t not wanting/not having the default of 10
- `member-ordering` ([source](https://palantir.github.io/tslint/rules/member-ordering/))
  - reason: the ordering was ` "static-field",  "instance-field", "static-method",  "instance-method"` which I can't agree with, if you want I can insert another order?

Do you have any input on the rules?
Any more rules you'd like to see? For example we could enforce or disable `public` for methods and field or we could enforce to always have `types` on methods.

ToDo's from your site (before merge):

- [x] enable Travis for `ark-mobile`

### CODE REVIEW

File: src/pages/transaction/transaction-show/transaction-show.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ export class TransactionShowPage {
 
   public transaction: Transaction;
   public currentNetwork: Network;
-  public equivalentAmount: number = 0;
+  public equivalentAmount = 0;
```
Comment (luciorubeens) [inline src/pages/transaction/transaction-show/transaction-show.ts]: Why did you remove the type from some properties, but keep others? 🤔

**Comment by luciorubeens** [comment]
What was the criteria for choosing these rules for tslint?

**Comment by Nasicus** [comment]
These are (almost) the default rules which come when you create a new project with `angular-cli`.

File: src/pages/transaction/transaction-show/transaction-show.ts (hunk 1 above)
Comment (Nasicus) [reply src/pages/transaction/transaction-show/transaction-show.ts]: I think the rule is, if the type is clear when declaring the variable, i.e. has an immediate value (like `0`) the type has to be omitted, else it must be declared.

**Comment by luciorubeens** [comment]
Very good, thanks! 👍🏻
